src/connector/src/source/kafka/source/reader.rs

```diff
@@ -73,21 +73,13 @@ impl SplitReader for KafkaSplitReader {
         properties.common.set_security_properties(&mut config);
         properties.set_client(&mut config);
 
-        // rdkafka fetching config
-        properties.rdkafka_properties.set_client(&mut config);
-
-        if config.get("group.id").is_none() {
-            config.set(
-                "group.id",
-                format!(
-                    "consumer-{}",
-                    SystemTime::now()
-                        .duration_since(UNIX_EPOCH)
-                        .unwrap()
-                        .as_micros()
-                ),
-            );
-        }
+        config.set(
+            "group.id",
+            format!(
+                "rw-consumer-{}-{}",
+                source_ctx.source_info.fragment_id, source_ctx.source_info.actor_id
```

Comment:
🤔 IIUC, the consumers of multiple parallelism should have same `group.id`?

Reply:
They will share the same source id rather than actor id

Reply:
> If we use the specific actor id rather than timestamp id, does it means the consume group previrous created will be resue?

Yes, the requirement is that a consumer group cannot be shared within parallelism. But the actor ids within the parallelism are unique.